CONTRIBUTING.md

```diff
@@ -344,9 +344,59 @@ Details for each point and good commit message examples can be found on https://
+### Baseline expectations from committers
+
+Both module and project committers must demonstrate technical mastery and commitment to the project’s community and growth.  In particular, they must show evidence that they are aligned with the goals outlined in [ARCHITECTURE](ARCHITECTURE.md), and demonstrate kindness and professionalism; help others in the project; and work collaboratively as a team.
+
+### Module committers
+#### What is a module committer?
+
+A module committer is a committer who has an entry associated with a module or folder in the project’s [`CODEOWNERS`](CODEOWNERS)  file.
+
+See [`CODEOWNERS`](CODEOWNERS) for modules, connectors, and libraries that lack active module committership. If you have interest in contributing to one of these, work toward becoming a commiter for that area.
+
+#### Expectations of a module committer
+
+Module committers have demonstrated mastery in one particular area of the project.  Some examples include:
+
+* A connector or plugin;
+* A part of the Presto codebase, such as the optimizer;
+* An external component, such as the Java client, or UI.
+
+In addition to technical mastery, they have demonstrated the values of the project through code reviews, design reviews, or responses to questions.  Examples should include many of the following:
+
+* The applicant is known to frequently review pull requests corresponding to the module they are applying for.
+* The applicant helps to maintain the module they are applying for when appropriate, such as fixing test cases, adding documentation, fixing bugs, and mentoring others.
+* The applicant is known to answer questions on Slack periodically.
+* The applicant has provided high quality feedback on Github issues and RFCs.
+
+### Project committers
+
+#### What is a project committer?
+
+A project committer is a committer who has access to approve code across the whole project by membership in the [committers](https://github.com/orgs/prestodb/teams/committers) Github team.
+
+#### Expectations of a project committer
+
+In addition to demonstrating mastery of at least one area of the codebase by becoming a module committer, they have also demonstrated the following:
+
+* They have contributed at least one non-trivial change to the project outside of their core area;
+* They exercise great judgment (including deferring to others when appropriate);
+* They have a curiosity of the codebase (by making code changes or reviewing code outside of their core area);
```

Comment:
have a curiosity of the --> are curious about

Frankly, I'd prefer to avoid assigning motivations to people. Either drop this one, or make it more concrete as simply ""They have made code changes or reviewed code in more than one area."

Reply:
I think this is meant to express that someone will look into areas that are unfamiliar to them (e.g learn more about some feature to help with review, or that they make a change in the way that is most maintainable/aligned with the overall architecture and needs of the project even if it requires delving into more unfamiliar territory vs. what's the minimal amount they need to understand to hack some solution together).

Reply:
I thought about this, and while "curiosity" does capture this well, I also agree that it's subjective (we don't want people to think if one is "curious enough").

So I broke it into two bullets, firstly, there are code changes and reviews outside their core area, and secondly, they set a high bar for their own code and for others during code review.  I think the combination of these two things will encapsulate what I intended to convey with that.

Reply:
makes sense. Did you want to delete this bullet point then? I still find the phrasing "a curiosity of the codebase" a little awkward.

Reply:
Oops, fixed!